portal/static/js/src/admin.js

```diff
@@ -187,6 +189,12 @@ import {EPROMS_MAIN_STUDY_ID, EPROMS_SUBSTUDY_ID} from "./data/common/consts.js"
                 $("#exportReportContainer").removeClass("open").popover("show");
                 $("#btnExportReport").attr("disabled", true);
                 $(".exportReport__status").addClass("active");
+                this.clearExportReportTimeoutID();
+                this.exportReportProgressTime = new Date();
+                let pastInfo = this.getCacheReportInfo();
+                if (pastInfo) {
+                    $(".exportReport__history").html(`<a href="${pastInfo.url}" target="_blank">View last result exported on ${tnthDates.formatDateString(pastInfo.date, "iso")}</a>`);
```

Comment:
```
View last result exported on ${tnthDates.formatDateString(pastInfo.date, "iso")}</a>`);
```
Do we need to make this string translatable? I'm not sure if site staff view adherence reports